server/opts_test.go

```diff
@@ -3475,3 +3475,117 @@ func TestProcessConfigString(t *testing.T) {
 		})
 	}
 }
+
+func TestAuthorizationTimeoutConfigParsing(t *testing.T) {
+	type testCase struct {
+		name                string
+		config              string
+		expectParsed        float64
+		expectRunning       float64
```

Comment:
This nuance is due to the fact that the config parser produces a certain value, but after the server is started using those values, the running option value could be different.